ui-modules/blueprint-composer/app/components/not-ready-appliance/not-ready-appliance.template.html

```diff
@@ -0,0 +1,36 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+<i uib-popover-template="'blueprint-composer/component/not-ready-appliance/not-ready-appliance.html'"
+   popover-is-open="warningIconClicked"
+   popover-placement="right"
```

Comment:
does it works well when you're on the boundaries of the designer? Does it change side automatically?

Reply:
It adapts the popover but always keeps it on the right. At some point the text is not readable. Thanks for this feedback, will check with my team about what to do